src/main/java/com/dev/movietheatre/model/MovieSession.java

```diff
@@ -16,7 +16,7 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @ManyToOne (fetch = FetchType.LAZY)
+    @ManyToOne
     @JoinColumn(name = "movie_id")
     private Movie movie;
     @ManyToOne (fetch = FetchType.LAZY)
```

Comment:
what about FetchType.LAZY for CinemaHall?

Reply:
deleted, done